src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlockMeasureCache.MeasureKey.cs

```diff
@@ -19,9 +21,11 @@ public class Comparer : IEqualityComparer<MeasureKey>
 			{
 				public static Comparer Instance { get; } = new Comparer();
 
-				public bool Equals(MeasureKey x, MeasureKey y)
+				public bool Equals(MeasureKey? x, MeasureKey? y)
 				{
-					var v = x._text == y._text
+					var v = x is not null
+						&& y is not null
+						&& x._text == y._text
```

Comment:
`Equals(null, null)` will return false. This is quite unexpected.

Reply:
That's true, though in this case, this comparer will never contain null keys (it would have NRE without the check anyways).

Reply:
I see the usage is `Comparer.Instance.Equals(this, key);`, so `x` is never null. But I think it's better idea (for possible future usages of this comparer) is to handle this case properly.